spec/characterization_services/pdf_characterization_service_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+require "valkyrie/derivatives/specs/shared_specs"
+
+RSpec.describe PDFCharacterizationService do
+  it_behaves_like "a Valkyrie::Derivatives::FileCharacterizationService"
+  # Required lets for shared specs.
+  let(:file) { fixture_file_upload("files/sample.pdf", "application/pdf") }
+  let(:resource) { FactoryBot.create_for_repository(:scanned_resource, files: [file]) }
+  let(:valid_file_set) { Wayfinder.for(resource).file_sets.first }
+  let(:persister) { ScannedResourcesController.change_set_persister.metadata_adapter.persister }
+  let(:file_characterization_service) { described_class }
+
+  before do
+    output = "547c81b080eb2d7c09e363a670c46960ac15a6821033263867dd59a31376509c"
+    ruby_mock = instance_double(Digest::SHA256, hexdigest: output)
+    allow(Digest::SHA256).to receive(:hexdigest).and_return(ruby_mock)
+  end
+
+  it "characterizes a sample file" do
+    file = fixture_file_upload("files/sample.pdf", "application/pdf")
+    resource = FactoryBot.create_for_repository(:scanned_resource, files: [file])
```

Comment:
Not sure if this is a strong stand against `let` or the shared specs were just added later, but `file` and `resource` are above so perhaps we should delete them here.

Reply:
Removed.